src/app/components/top/solicitudes/solicitudes.component.ts

```diff
@@ -669,6 +692,52 @@ export class SolicitudesComponent implements OnInit {
         }
     }
 
+    returnReferir(event) {
+        this.showAuditar = false;
+        this.showReferir = false;
+        this.showSidebar = false;
+        this.columns.splice(1, 0, { key: 'paciente', label: 'Paciente' });
+        if (!event.status) {
+            const statuses = ['pendiente', 'asignada', 'rechazada', 'referida'];
+            if (event.status !== this.prestacionSeleccionada.estados && this.prestacionSeleccionada.estados.length) {
+                let patch;
+
+                if (event.status !== 3) {
+                    patch = {
+                        op: 'estadoPush',
+                        estado: { tipo: statuses[event.status], observaciones: event.observaciones }
+                    };
+
+                    // DEPRECADO
+                    if (!event.status) {
+                        patch.motivoRechazo = event.observaciones;
+                    }
+                    if (event.prioridad) {
+                        patch.prioridad = event.prioridad;
+                    }
+                    if (event.profesional) {
+                        patch.profesional = event.profesional;
+                    }
+
+                } else {
+                    patch = {
+                        op: 'referir',
+                        estado: event.estado,
+                        observaciones: event.observaciones,
+                        organizacion: event.organizacion,
+                        profesional: event.profesional,
+                        tipoPrestacion: event.prestacion
+                    };
+                }
+                this.servicioPrestacion.patch(this.prestacionSeleccionada.id, patch).subscribe(
+                    respuesta => {
+                        this.cargarSolicitudes();
```

Comment:
Ya que respuesta no se utiliza se puede dejar el patch como:
this.servicioPrestacion.patch(this.prestacionSeleccionada.id, patch).subscribe(() => {
    this.cargarSolicitudes();
});

Reply:
Corregido!